lib/tasks/cucumber.rake

```diff
@@ -41,10 +53,12 @@ begin
       ::CodeStatistics::TEST_TYPES << "Cucumber features" if File.exist?('features')
     end
   end
-  desc 'Alias for cucumber:ok'
-  task :cucumber => 'cucumber:ok'
 
+  desc 'Alias for cucumber:ok'
   task :default => :cucumber
+  task :cucumber => 'cucumber:aa'
```

Comment:
I pulled your branch down and played with it.  Is there any way to change it so that if you do not specify `hq` or `aa` that it runs all the tests in the app instead of just `aa`?